constraintlayout/core/src/main/java/androidx/constraintlayout/core/state/WidgetFrame.java

```diff
@@ -587,4 +599,6 @@ void logv(String str) {
         System.out.println(ss + " " + str);
     }
 
+    public void setCustomValue(CustomAttribute valueAt, float[] mTempValues) {
+    }
```

Comment:
unused?

Reply:
Called but not implemented.  This is for custom Attribute support in the Compose